meta-mbl-distro/classes/mbl-partitions.bbclass

```diff
@@ -24,138 +24,407 @@ inherit mbl-artifact-names
 # will be used in your case.
 MBL_FLASH_ERASE_BLOCK_SIZE_KiB ?= "16384"
 
+MBL_DEFAULT_ALIGN_KiB ?= "${MBL_FLASH_ERASE_BLOCK_SIZE_KiB}"
+MBL_DEFAULT_FSTYPE ?= "ext4"
+MBL_DEFAULT_MOUNT_OPTS ?= "defaults"
+MBL_DEFAULT_FS_FREQ ?= "0"
+MBL_DEFAULT_FS_PASSNO ?= "0"
 
+# ------------------------------------------------------------------------------
+# Partition variables
+# ------------------------------------------------------------------------------
 # We have two different flavours of "partition" in MBL:
 # * Normal file system partitions, recorded in the partition table.
 # * Raw non-file system "partitions", not recorded in the partition table.
+# 
+# List of all partitions, in order
+MBL_PARTITION_NAMES = ""
+#
+# For each of these "partitions" there are the following variables:
```

Comment:
Which "partitions"?
The line above is setting it to empty.
Maybe we need a better comment about what are the default/expected/supported partition layout. Or a specific documentation file to give a better background.

Reply:
I've partially addressed this with https://github.com/ARMmbed/meta-mbl/pull/617/commits/80acca9bf0501943da70a460f557e8d80dca454d.

There will be documentation (in the docs repo) about the partition layout and how it can be adjusted (https://jira.arm.com/browse/IOTMBL-531).